finetune/base_models/gpt/featurizer.py

```diff
@@ -19,7 +19,7 @@ def mask_attn_weights(w):
 
 def mask_pad(w, lengths):
     batch = shape_list(lengths)[0]
-    maxlen = tf.maximum(lengths)
+    maxlen = tf.cast(tf.reduce_max(lengths), tf.int32)
```

Comment:
what's the difference in behavior? I'm having trouble learning about the differences online